src/Features/CSharp/Portable/InlineHints/CSharpInlineTypeHintsService.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.InlineHints;
+
+namespace Microsoft.CodeAnalysis.CSharp.InlineHints
+{
+    [ExportLanguageService(typeof(IInlineTypeHintsService), LanguageNames.CSharp), Shared]
+    internal class CSharpInlineTypeHintsService : AbstractInlineTypeHintsService
+    {
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public CSharpInlineTypeHintsService()
+        {
+        }
+
+        protected override InlineTypeHint? TryGetTypeHint(
+            SemanticModel semanticModel,
+            SyntaxNode node,
+            bool forImplicitVariableTypes,
+            bool forLambdaParameterTypes,
+            CancellationToken cancellationToken)
+        {
+            if (forImplicitVariableTypes)
+            {
+                if (node is VariableDeclarationSyntax variableDeclaration &&
```

Comment:
This might be intentional, but perhaps look for VariableDeclarator instead, to pick up some more cases?
![image](https://user-images.githubusercontent.com/754264/95424643-a9943300-098e-11eb-9584-6cb3dfc64496.png)

Reply:
this is handled below now (with tests).